### ISSUE TITLE
Feature request: `devbox shell -p somepackage`

### ISSUE THREAD

**Issue by robrich** [jetpack-io/devbox#50]
It makes it feel a lot like [`npx`](https://www.npmjs.com/package/npx). Nothing to install or configure, I just run an isolated command and get results.

**Comment by morey-tech** [comment]
I think something like `devbox run <package>` could be a better syntax. It follows that of `docker`. The `run` signifies that `devbox` will execute the isolated command and exit. As opposed to `shell`, which given its current function, could imply that it will stay active after the command has been completed (i.e., a way to install a package in a shell without defining it in the config).

**Comment by Lagoja** [comment]
We're not totally set on how the CLI will do it, but we are looking into `devbox run` like functionality for an upcoming release. See #94 for more details and discussion

**Comment by Lagoja** [comment]
This is now available in Devbox 0.0.7! You can run a one off command in your devbox shell using:

```bash
devbox shell -- <cmd>
````

This will launch a shell based on your devbox.json, run your command, and then exit back to your normal shell. For example:

```bash
~ $  devbox shell -- echo "Hello World"
Installing nix packages. This may take a while...
Starting a devbox shell...
Hello World
~ $ 
```
More info is in our new CLI Docs: https://www.jetpack.io/devbox/docs/cli_reference/devbox_shell/
